### PR TITLE
UINV-322 - Select Upload location type in settings

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UINV-322](https://issues.folio.org/browse/UINV-322)
## Approach

- add Port, directory, and location type fields
- hide Schedule exports filed temporarily due to discussion with PO

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationForm.js [hunk 1]
```diff
@@ -159,6 +175,35 @@ const BatchGroupConfigurationForm = ({
...
+            xs={4}
+          >
+            <Field
+              id="ftpPort"
+              label={<FormattedMessage id="ui-invoice.settings.batchGroupConfiguration.port" />}
+              name="ftpPort"
+              component={TextField}
+              fullWidth
+              type="number"
+              parse={v => v}
```
Comment (NikitaSedyx) [inline src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationForm.js:191]: I think it's not required

File: src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationForm.js [hunk 2]
```diff
@@ -159,6 +175,35 @@ const BatchGroupConfigurationForm = ({
...
+            data-test-col-directory
+            xs={4}
+          >
+            <Field
+              id="uploadDirectory"
+              label={<FormattedMessage id="ui-invoice.settings.batchGroupConfiguration.directory" />}
+              name="uploadDirectory"
+              component={TextField}
+              fullWidth
+              parse={v => v}
```
Comment (NikitaSedyx) [inline src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationForm.js:204]: same

File: src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationForm.js (hunk 1 above)
Comment (alisher-epam) [reply src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationForm.js:191]: @NikitaSedyx you mean `parse={v => v}`?

Comment (NikitaSedyx) [reply src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationForm.js:191]: yes

**Review by NikitaSedyx — APPROVED** [review]

**Review by usavkov-epam — APPROVED** [review]

**Pull request merged**